db/fixtures/miq_product_features.yml

```diff
@@ -2828,10 +2828,6 @@
       :description: Edit Visuals
       :feature_type: admin
       :identifier: my_settings_visuals
-    - :name: Default Views
-      :description: Edit Default Views
-      :feature_type: admin
-      :identifier: my_settings_default_views
```

Comment:
I think this just works but wanted to ask if there are existing roles with this feature, do they automatically get this removed (and not orphaned).  Can you try giving this role to a user role on master, then switch to this branch, seed this fixture via `rake db:seed` and verify that user role can navigate around without any errors in the related my settings pages?

Reply:
Yeah I will test it out

Reply:
Yeah it seems to be working. Just tested this out and everything seems to be working fine.